cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -34,6 +34,80 @@ API Changes
    `CDAP-1078 <https://issues.cask.co/browse/CDAP-1078>`__).
 
 
+New Features
+------------
+
+- **Spark**
+
+  - Spark now uses a configurer-style API for specifying
+  - Spark can now run as a part of a Workflow
+
+- **Security**
+
+  - CDAP Master now obtains and refreshes Kerberos tickets programmatically
+
+- **Datasets**
+
+  - A new, experimental dataset type to support time-partitioned file sets has been added
+  - Time-partitioned file sets can be queried with Impala on CDH distributions
+  - Streams can be made queryable with Impala by deploying an adapter that periodically
+    converts it into partitions of a time-partitioned file set
+  - Support for different levels of conflict detection: ``ROW``, ``COLUMN``, or ``NONE``
+  - Removed support for ``@DisableTransaction``
+  - Support for annotating a Stream with a schema
+  - A new API for uploading entire files to a Stream has been added 
+
+- **Workflow**
+
+  - Workflow now uses a configurer-style API for specifying
+  - Multiple instances of a Workflow can be run concurrently
+  - Programs are no longer part of a Workflow; instead, they are added in the application
+    and are referenced a Workflow using their names
+  - Schedules are now at the application level and properties can be specified for
+    a Schedules; these properties will be passed to the scheduled program as runtime
+    arguments
+
+Known Issues
+------------
+- See also the *Known Issues* of `version 2.6.1. <#known-issues-261>`_
+- When upgrading an existing CDAP installation to 2.7.0, all metrics are reset.
```

Comment:
@awholegunch Can we add this as an known issue:

* Cannot change the TTL of Streams

@chtyim Does this look ok?